helpers/mime.js

```diff
@@ -2,20 +2,46 @@
 
 const mime = require('mime-types');
 
-const FORCE_EXTENSION_MAPPING = {
-    yidf: 'txt',
-    state: 'txt',
-    diff: 'txt',
-    xml: 'txt' // FIXME: Chrome is not displaying xml files
-};
+const KNOWN_FILE_NAMES_IN_TEXT_FORMAT = ['dockerfile', 'makefile'];
+const KNOWN_FILE_EXTS_IN_TEXT_FORMAT = [
```

Comment:
This is what we have to avoid, this list will only grow on file types and is not a good solution. Please revisit the code and try an alternate approach

Reply:
Hmm, not sure what are the alternatives?

Reply:
Since we changed from displaying files by default, https://github.com/screwdriver-cd/store/pull/114/files#diff-689aa9b3faaaff60458d1d0ef6aaebbb965c9ed5bc6f43cf647a1ef6354f0605R67

I thought the proper way to display files is to explicitly list out the `file name`s and `file extension`s as displayable. We would need to have a list for file ends with `yidf` (customized), `tox` (python), `.d.ts` (typescript). These are not as popular as `txt, zip, rar ` but heavily used.

I hope there's a better way not to maintain the growing list ourself.

Reply:
why do we need to display all these files ?